### PR TITLE
Migrate existing settings to new Design

### PR DESCRIPTION
**Pull request by zeitschlag**
This is the first part for IOS-14. It migrates the existing settings over to new design. And it looks like this:

### CODE REVIEW

File: Mastodon/Scene/Settings/Notification Settings/Cells/NotificationSettingsDisabledTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        backgroundColor = Asset.Colors.Brand.blurple.color.withAlphaComponent(0.15)
+
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    private func setupConstraints() {
+        let constraints: [NSLayoutConstraint] = [
+            appBadgeImageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16),
```
Comment (DK1DA) [inline Mastodon/Scene/Settings/Notification Settings/Cells/NotificationSettingsDisabledTableViewCell.swift]: I feel like we should offload the `8` and `16` into some constants enum etc 😅

Comment (DK1DA) [reply Mastodon/Scene/Settings/Notification Settings/Cells/NotificationSettingsDisabledTableViewCell.swift]: But we can do this later, no need to touch all those places again right now.

File: Podfile [hunk 2]
```diff
@@ -17,8 +17,6 @@ target 'Mastodon' do
   pod 'Kanna', '~> 5.2.2'
   pod 'Sourcery', '~> 1.9'
 
-  # DEBUG
```
Comment (DK1DA) [inline Podfile:20]: 👍

File: MastodonSDK/Sources/CoreDataStack/Entity/App/Setting.swift [hunk 3]
```diff
@@ -14,6 +14,7 @@ public final class Setting: NSManagedObject {
     @NSManaged public var userID: String
 
 //    @NSManaged public var appearanceRaw: String
```
Comment (DK1DA) [inline MastodonSDK/Sources/CoreDataStack/Entity/App/Setting.swift]: Shouldn't we get rid of this as well?

**Review by kimar — APPROVED** [review]
LGTM 👍  Very nice

File: MastodonSDK/Sources/CoreDataStack/Entity/App/Setting.swift (hunk 3 above)
Comment (zeitschlag) [reply MastodonSDK/Sources/CoreDataStack/Entity/App/Setting.swift]: Oh, yeah, you're right. Good point! 👍

**Pull request merged**